packages/@lwc/engine-server/src/apis/render-component.ts

```diff
@@ -7,9 +7,10 @@
 import { createVM, connectRootElement, LightningElement } from '@lwc/engine-core';
 import { isString, isFunction, isObject, isNull } from '@lwc/shared';
 
-import { createElement } from '../renderer';
+import * as renderer from '../renderer';
 import { serializeElement } from '../serializer';
 import { HostElement, HostNodeType } from '../types';
+import type { RendererAPI } from '@lwc/engine-core/types/renderer';
```

Comment:
We shouldn't have to drill into the packages. This type should be exported from `@lwc/engine-core`.